recipes/datatable/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -euo pipefail
+
+{
+
+cat >fake-git <<'EOF'
+#!/bin/bash
+
+if [ "$1" == "show" ]; then
+  date +%s
+else
+  echo "FAKE_BUILD_INFO_FOR_CONDA"
+fi
+EOF
+
+chmod +x fake-git
+mv fake-git git
+echo 0.11.0 >VERSION.txt
```

Comment:
In the `ci/ext.py` file it says: 
```
# - When building from source distribution (file VERSION.txt is
#   absent, the version is taken from datatable/_build_info.py) file;
```
So I do not think this part is necessary.

Reply:
The build fails without this line.

Reply:
Actually, maybe I should try again. I think it might have been a test failure that is now fixed by excluding that test from the pytest call.

Reply:
Right. I think this was a fix for what seems to be a “bug” in ext.py that prevents building from the source tarball. Removing it caused the latest Linux build to fail. I’ll try adding it back in later to get it to work. Unless you know a better solution?